### ISSUE TITLE
Message input bar not visible

### ISSUE THREAD

**Issue by keyhan76** [MessageKit/MessageKit#1343]
Message Input bar is not visible when using a container view for the message view controller.
By container view I mean using the message view controller inside a container view so it would be a child view controller.

**Comment by martinpucik** [comment]
Hey @keyhan76 could you try to look at our example project [here](https://github.com/MessageKit/MessageKit/blob/master/Example/Sources/View%20Controllers/MessageContainerController.swift) and try to reproduce your issue?

**Comment by keyhan76** [comment]
@martinpucik  Yes I checked it. But my issue is that I’m using container view in Storyboard and in this case the child view controller is directly inherited from MessageViewController.

**Comment by martinpucik** [comment]
Well, make sure you're handling the `inputAccessoryView` in your container VC properly, as we're in out example project

https://github.com/MessageKit/MessageKit/blob/417722060ead863b93b94fab7c5eedd4e1f6db53/Example/Sources/View%20Controllers/MessageContainerController.swift#L45-L53

**Comment by keyhan76** [comment]
@martinpucik yes I have added these methods two.
For the canBecomeFirstResponder I have returned true.
For the inputAccessoryView I have returned messageInputBar

Is that right?

**Comment by keyhan76** [comment]
@martinpucik I fixed this problem, but now I'm facing another problem:)
I can't customize the input bar accessory view like the one in advancedExample.
I've copied all the functions but I just get the default style of input view.

**Comment by martinpucik** [comment]
@keyhan76 that looks like you're modifying different `inputAccessoryView`. Can you compare their addresses?

**Comment by avikhemani** [comment]
@keyhan76 how did you fix the initial problem? I'm running into the same issue :/ my inputAccessoryView is also not showing in a container view

**Comment by keyhan76** [comment]
@avikhemani just remove the container view and made it exactly like the example project

**Comment by Omarhd** [comment]
/// Required for the `MessageInputBar` to be visible 
 override var canBecomeFirstResponder: Bool { 
     return conversationViewController.canBecomeFirstResponder 
 } 
  
 /// Required for the `MessageInputBar` to be visible 
 override var inputAccessoryView: UIView? { 
     return conversationViewController.inputAccessoryView 
 } 
![Simulator Screenshot - iPhone 15 - 2024-04-06 at 09 46 16](https://github.com/MessageKit/MessageKit/assets/39955622/51d83833-dec2-4e1a-b7a0-4c5f9d1ff00c)

 
 these override vars added input bar for all My ChidViews not just my chat ViewController
 
 basically I have container view that contains a PageController and Page Holds 5 ViewControllers
